README.md

```diff
@@ -388,6 +388,9 @@ productType,name.en,variantId
     --categoryBy                  Define which identifier should be used to for the categories column - either slug or externalId. If nothing given the named path is used.
     --filterVariantsByAttributes  Query string to filter variants of products
     --filterPrices  Query string to filter prices of variants
+    --templateDelimiter <delimiter> Delimiter used in template | default: ,
+    --cellDelimiter <delimiter>   Delimiter used to sepparate cells in output file | default: ,
```

Comment:
it is because templates are now created with "," delimiter which is default but for example excel is expecting ";" - so thats why we have input and output delimiter .. there should be also third one which would specify delimiter of multiple values in one cell but I did not find any way how to specify it in CSV module which we are using

about naming convention: what about "inDelimiter" and "outDelimiter" ?